src/Init/Init_Load_Parameter.cpp

```diff
@@ -504,7 +510,7 @@ void Init_Load_Parameter()
    ReadPara->Add( "OPT__CK_FINITE",             &OPT__CK_FINITE,                  false,           Useless_bool,  Useless_bool   );
    ReadPara->Add( "OPT__CK_PATCH_ALLOCATE",     &OPT__CK_PATCH_ALLOCATE,          false,           Useless_bool,  Useless_bool   );
    ReadPara->Add( "OPT__CK_FLUX_ALLOCATE",      &OPT__CK_FLUX_ALLOCATE,           false,           Useless_bool,  Useless_bool   );
-#  if ( MODEL == HYDRO )
+#  if ( MODEL == HYDRO && defined SRHD )
```

Comment:
`OPT__CK_NEGATIVE` is not assigned if `MODEL == HYDRO && !defined SRHD` which might cause some problems at `src/Auxiliary/Aux_Check.cpp` at line 39 and `src/Aux_TakeNote.cpp` at line 1461.